src/components/modal/loginModal/loginModaPresenterl.tsx

```diff
@@ -25,7 +25,7 @@ const LoginModal: React.FC = () => {
   const [isLoading, setIsLoading] = useRecoilState(loadingState);
 
   const [setPassword, tryLogin] = useLogin(closeModal, setIsLoading);
-  const tryGoogleLogin = googleLogin(closeModal, setIsLoading);
+  const tryGoogleLogin = useGoogleLogin(closeModal, setIsLoading);
 
   const checkEnter = (e: { key: string }) => {
```

Comment:
```suggestion
  const checkEnter = ({key}: { key: string }) => {
```
비구조화문법으로 인자값을 받아내는 것은 어떨까요?
그리고 key보다는 input이 어울린다 생각합니다.

Reply:
key 이벤트를 특정하는 로직이라 input 보다는 key 가 맞다고 생각됩니다.